kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/OASModelFilter.java

```diff
@@ -28,6 +29,24 @@ public class OASModelFilter implements OASFilter {
     private static final String SECURITY_SCHEME_NAME_OAUTH = "Bearer";
     private static final String SECURITY_SCHEME_NAME_BASIC = "BasicAuth";
 
+    @Override
+    public Schema filterSchema(Schema schema) {
+        List<Schema> allOf = schema.getAllOf();
+
+        // Remove superfluous `nullable: false` added by OpenAPI scanner
+        if (allOf != null && allOf.stream().anyMatch(s -> Boolean.FALSE.equals(s.getNullable()))) {
```

Comment:
FYI - I see this code being very useful to fix general inheritance issues for OpenAPI schema. 
Do you think it is possible to have `openapi-helpers` or best practices docs on upstream level so we can reuse in other projects like Smart events: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/861

For now we can copy paste this but it is really good pattern and it might be hard for anyone to figure it out themselves.

Reply:
Just opened this: https://github.com/smallrye/smallrye-open-api/issues/1180

Do you think besides fixing this in the scanner that we need anything else?

Reply:
That should be enough. Title breaking generator is an bug on the generator side.